deps.ts

```diff
@@ -1,26 +1,27 @@
 export {
   assert,
+  assertEquals,
   assertThrowsAsync,
 } from "https://deno.land/std@0.74.0/testing/asserts.ts";
+
+export type { Payload } from "https://deno.land/x/djwt@v2.0/mod.ts";
+export type { AlgorithmInput } from "https://deno.land/x/djwt@v2.0/algorithm.ts";
 export {
-  JwtObject,
-  JwtValidation,
-  validateJwt,
-  Validation,
-} from "https://deno.land/x/djwt@v1.7/validate.ts";
-export {
-  Algorithm,
-  Jose,
-  makeJwt,
-  Payload,
-  setExpiration,
-} from "https://deno.land/x/djwt@v1.7/create.ts";
+  create,
+  getNumericDate,
+  verify,
+} from "https://deno.land/x/djwt@v2.0/mod.ts";
+
 export { createHttpError } from "https://deno.land/x/oak@v6.3.1/httpError.ts";
-export {
-  Context,
+
+export type {
```

Comment:
Is there a reason for why you split these imports up in types and non types?

Reply:
Yes! After the `--isolatedModules` flag was enabled by default on Deno (after version 1.5) this throws errors. It's now mandatory to use import and export types.

Reply:
Ah, yes, true... It has been a minor version since I dealt with Deno 😅